diff --git a/node_modules/classic-level/binding.js b/node_modules/classic-level/binding.js
index ffd9386..11f662b 100644
--- a/node_modules/classic-level/binding.js
+++ b/node_modules/classic-level/binding.js
@@ -1 +1 @@
-module.exports = require('node-gyp-build')(__dirname)
+module.exports = require('node-gyp-build')(typeof __dirname !== 'undefined' ? __dirname : process.cwd() + '/node_modules/classic-level')
